mqtt-v5-broker/src/broker.rs

```diff
@@ -45,14 +55,33 @@ struct Session {
     will: Option<FinalWill>,
 }
 
+impl std::fmt::Debug for Session {
```

Comment:
Was `pub client_sender: Option<Sender<ClientMessage>>,` giving you trouble?

I'd prefer to avoid a manual `Debug` impl which is only working around one particular field, probably better to fix the field itself with a new-type, or see if there's an alternative which _does_ impl `Debug`.